semcore/date-picker/src/components/PickerAbstract.jsx

```diff
@@ -104,7 +117,7 @@ class PickerAbstract extends Component {
       return isDisabledDay ? null : current_day;
     };
 
-    if (e.key === 'Space' || (e.key === 'Enter' && highlighted.length)) {
+    if (place === 'popper' && (e.key === 'Space' || (e.key === 'Enter' && highlighted.length))) {
```

Comment:
Why don't we handle ArrowUp to close the popper?

Reply:
Because when we on calendar we use ArrowUp to move by the days.